components/hab/src/command/pkg/bulkupload.rs

```diff
@@ -48,13 +48,8 @@ pub fn start(ui: &mut UI,
              auto_create_origins: bool,
              key_path: &Path)
              -> Result<()> {
-    const OPTIONS: glob::MatchOptions = glob::MatchOptions { case_sensitive:              true,
-                                                             require_literal_separator:   true,
-                                                             require_literal_leading_dot: true, };
-    let artifact_paths =
-        vec_from_glob_with(&artifact_path.join("*.hart").display().to_string(), OPTIONS);
-
-    let pub_keys_paths = vec_from_glob_with(&key_path.join("*.pub").display().to_string(), OPTIONS);
+    let artifact_paths = paths_with_extension(artifact_path, "hart");
+    let pub_keys_paths = paths_with_extension(key_path, "pub");
```

Comment:
![](https://media.giphy.com/media/6utXdQXMEYKLETAe9U/giphy.gif)

Reply:
Took it one step further and used the `PUBLIC_KEY_SUFFIX` constant in https://github.com/habitat-sh/habitat/pull/7279/commits/3294046fb9b093ce7fb0f88eb8ec1d516366f83f 😄